fixtures/martin/mac-aura-.json

```diff
@@ -0,0 +1,518 @@
+  "physical": {
+    "dimensions": [302, 360, 302],
+    "weight": 5.6,
+    "power": 260,
+    "DMXconnector": "5-pin",
+    "bulb": {
+      "type": "LED",
+      "colorTemperature": 10000,
+      "lumens": 3850
+    },
+    "lens": {
+      "name": "beam&aura",
+      "degreesMinMax": [11, 58]
+    },
+    "focus": {
+      "type": "Fixed",
+      "panMax": 540,
+      "tiltMax": 232
+    }
+  },
+  "availableChannels": {
+    "shutter": {
+      "type": "Shutter",
+      "capabilities": [
+        {
+          "range": [0, 19],
+          "name": "shutter closed"
+        },
+        {
+          "range": [20, 24],
+          "name": "shutter open "
+        },
+        {
+          "range": [25, 64],
+          "name": "strobe 1 (fast>slow)"
+        },
+        {
+          "range": [65, 69],
+          "name": "shutter open"
+        },
+        {
+          "range": [70, 84],
+          "name": "strobe 2: opening pulse (fast>slow)"
+        },
+        {
+          "range": [85, 89],
+          "name": "shutter open"
+        },
+        {
+          "range": [90, 104],
+          "name": "strobe 3: closing pulse (fast>slow)"
+        },
+        {
+          "range": [105, 109],
+          "name": "shutter open"
+        },
+        {
+          "range": [110, 124],
+          "name": "strobe 4: random strobe (fast>slow)"
+        },
+        {
+          "range": [125, 129],
+          "name": "shutter open"
+        },
+        {
+          "range": [130, 144],
+          "name": "strobe 5: random opening pulse (fast>slow)"
+        },
+        {
+          "range": [145, 149],
+          "name": "shutter open"
+        },
+        {
+          "range": [150, 164],
+          "name": "strobe 6: random closing pulse (fast>slow)"
+        },
+        {
+          "range": [165, 169],
+          "name": "shutter open"
+        },
+        {
+          "range": [170, 184],
+          "name": "strobe 7: burst pulse (fast>slow)"
+        },
+        {
+          "range": [185, 189],
+          "name": "shutter open"
+        },
+        {
+          "range": [190, 204],
+          "name": "strobe 8: random burst pulse (fast>slow)"
+        },
+        {
+          "range": [205, 209],
+          "name": "shutter open"
+        },
+        {
+          "range": [210, 224],
+          "name": "strobe 9: sine wave (fast>slow)"
+        },
+        {
+          "range": [225, 229],
+          "name": "shutter open"
+        },
+        {
+          "range": [230, 244],
+          "name": "strobe 10: burst (fast>slow)"
+        },
+        {
+          "range": [245, 255],
+          "name": "shutter open"
+        }
+      ]
+    },
+    "beam dimmer": {
+      "type": "Intensity",
+      "capabilities": [
+        {
+          "range": [0, 255],
+          "name": "beam dimmer (0>100)"
+        }
+      ]
+    },
+    "zoom": {
+      "type": "[add-value]",
```

Comment:
@novasfronteiras It seems that we still have a bug in the fixture editor that didn't catch your new type here. What did you enter here?